relay-server/src/actors/project_buffer.rs

```diff
@@ -302,6 +306,24 @@ impl BufferService {
         Ok(size)
     }
 
+    /// Returns duration since UNIX_EPOCH.
+    fn now() -> Duration {
+        SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap_or_default()
+    }
+
+    /// Returns the timestamp in form of fixed byte array.
+    ///
+    /// Timestamp represents the point in time before the provided duration.
```

Comment:
I think this is easier to understand:
```suggestion
    /// Returns the timestamp before the provided duration, in form of fixed byte array.
```